ocp_utilities/infra.py

```diff
@@ -79,6 +82,45 @@ def assert_pods_failed_or_pending(pods):
         )
 
 
+def assert_nodes_in_healthy_condition(nodes):
+    """
+    Validates nodes are in a healthy condition.
+    None of the following node conditions is True:
+        - DiskPressure
+        - MemoryPressure
+        - PIDPressure
+        - NetworkUnavailable
+
+    Args:
+         nodes(list): List of Node objects
```

Comment:
Hows the list being passed to this and other fns

Reply:
It's passed from cluster_sanity in managed-services-integration-tests/tests/cluster_sanity
```
@pytest.mark.smoke
def test_cluster_sanity(nodes, pods, junitxml_plugin):
    cluster_sanity(
        nodes=nodes,
        pods=pods,
        junitxml_property=junitxml_plugin,
    )
```
`cluster_sanity` the calls the calls `assert_nodes_in_healthy_condition `